integration/src/integration/java/pl/allegro/tech/hermes/integration/IntegrationTest.java

```diff
@@ -35,4 +36,9 @@ public void initializeIntegrationTest() {
                 CONFIG_FACTORY);
     }
 
+    @AfterMethod
+    public void after() {
+        management.query().querySubscriptions("{\"query\": {}}").forEach(sub ->
```

Comment:
why do we need to do this actually? isn't it some kind of an asynchronous trap, where we think that subs have been removed, but we don't wait for them to be actually stopped/removed on consumers side, so we basically don't know and it might bring some unexpected behavior